### DISCUSSION TITLE
tailwind and postcss duplicate files

### DISCUSSION THREAD

**Discussion by SebasCC99** [tailwindlabs/tailwindcss#9758] [Help] [answered]
Hello, Its my first time using Tailwind CSS, I followed the instructions for installing it using a framework, Im using Next JS 13.
I believe when I ran for the first time the dev server of Next, duplicate files were created. My file directory now looks like this.

<img width="291" alt="Screenshot 2022-11-06 at 10 20 46 PM" src="https://user-images.githubusercontent.com/65096739/200219634-79d34a15-6fc4-4d15-bfc9-7dec5ff213a1.png">

Im not sure what changed but Im not sure how this will affect my code. Ive tried searching online about this issue but I could not find anything about. 

- Is this expected behavior
- Should I add these files to the gitignore?

Thank you

**Comment by wongjn** [comment, marked as answer]
I do not believe this is expected behavior. You do not need these duplicate files at all – they shouldn't affect anything.

**Reply by SebasCC99** [reply]
Thank you! I'll delete them and see if doing something reproduces the issue
